### ISSUE TITLE
CLI does not work when NPM installed locally

### ISSUE THREAD

**Issue by frank-weindel** [rdkcentral/Lightning-CLI#95]
I'm not a fan of installing CLI tools globally when I can get away with not doing so. So I prefer installing them locally into my project and relying on NPM script commands or NPX to run those cli scripts. Obviously this is just a preference. The problem is, Lightning-CLI does not seem to work when installed locally. See command logs below:

```
fweind200@HQSML-1720180 tv-app % npx lng dist
 
✔ Removing "static" folder
 
✔ Copying static assets to "es6"
 
✖ Error while creating ES6 bundle (see log)
 

(node:51964) UnhandledPromiseRejectionWarning: TypeError: PromiseReject called on non-object
    at reject (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:51964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:51964) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

```
fweind200@HQSML-1720180 tv-app % npx lng dev 
 
✔ Removing "build" folder
 
✔ Ensuring "build" folder exists
 
✔ Copying support files to "build"
 
✔ Copying static assets to "build"
 
✔ Copying settings.json to "build"
 
✔ Copying metadata.json to "build"
 
✖ Error while creating ES6 bundle (see log)
 

✖ Building ES6 appBundle and saving to "build"
```
(Not sure where the log is)

These commands work fine with a globally installed Lightning-CLI.

**Comment by tapsboy** [comment]
Hello,

I also ran into the same issue.

While, I am also not a fan of installing globally, there is an additional issue where I would like to run the build in a CI pipeline, where installing a global CLI isn't an options. An ability to add the cli as a dev dependency will be awesome.

Thanks again for the excellent framework !!!

**Comment by michielvandergeest** [comment]
Hi @tapsboy (and @frank-weindel as well off course): we have identified the problem and a solution is in the works

**Comment by michielvandergeest** [comment]
this has been addressed in PR #102 which has just been merged into `dev`, the functionality will be available in the next release.

**Comment by michielvandergeest** [comment]
Local CLI install is now supported as of version 2.4.0.
